### DISCUSSION TITLE
gatsby-plugin-image with gatsby-strapi setup

### DISCUSSION THREAD

**Discussion by muscaiu** [gatsbyjs/gatsby#35389] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Summary

In the [`gatsby-plugin-image`](https://www.npmjs.com/package/gatsby-plugin-image#dynamic-images) documentation it's written we can access the image under this query:

```
    avatar {
      childImageSharp {
        gatsbyImageData(
          width: 200
          placeholder: BLURRED
          formats: [AUTO, WEBP, AVIF]
        )
      }
    }
```

But in my gatsby-strapy project the path contains a localFile looks a little different and i can't render the image:

```
export const query = graphql`
  {
    allStrapiProjects(filter: { featured: { eq: true } }) {
      nodes {
        title
        image {
          localFile {
            childImageSharp {
              gatsbyImageData
            }
          }
        }
    }
```

<img width="1440" alt="Screenshot 2022-04-10 at 08 56 41" src="https://user-images.githubusercontent.com/11426309/162604717-bd1a8578-fa56-4955-ac68-7aeb644e6b6e.png">

```
  const gatsbyImg = getImage(image);
  console.log("gatsbyImg:", gatsbyImg); // undefined
```

What is the correct way of accessing and rendering the image?

### Steps to Resolve this Issue

1.
2.
3.
...

**Comment by LekoArts** [comment]
The guide can only show the most common formats and thus specific plugins can have a slightly different schema. I recommend referring to https://github.com/strapi/gatsby-source-strapi documentation and ask specific questions there.
